src/administration/integrations/notifications.md

```diff
@@ -24,9 +26,41 @@ Health notifications can be set up via the [Platform.sh CLI](/gettingstarted/cli
 
 ### Email notifications
 
+#### Default email notifications
```

Comment:
I wouldn't put the defaults as a sub-section of emails.  For one, we try to avoid 4th level headers generally.

Rather, this should be a level 2 header, possibly even before "Available notifications", that calls out that projects get an email notification for admins upon creation.  Then defer the rest of the description of how email works to the email section, already in place.  Then if there's stuff to expand on (like the new #admins magic value), document those inline as a standard feature of whatever scope.

Reply:
Makes sense to me, thanks for explaining @Crell!